hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/BytesBuffer.java

```diff
@@ -45,8 +46,8 @@
     public static final int UINT16_MAX = ((short) -1) & 0xffff;
     public static final long UINT32_MAX = (-1) & 0xffffffffL;
 
-    public static final int ID_MAX_LEN = UINT8_MAX & 0x7f + 1; // 128
-    public static final int BIG_ID_MAX_LEN = UINT16_MAX & 0x7fff + 1; // 32768
+    public static final int ID_MAX_LEN = UINT8_MAX & 0x7f; // 127
+    public static final int BIG_ID_MAX_LEN = UINT16_MAX & 0x7fff; // 32767
```

Comment:
max length is 32767-256

Reply:
yes, 1byte id len must be in range [0, fe],  2byte id len must be in range [0, feff]